lgc/patch/ShaderMerger.cpp

```diff
@@ -596,9 +596,9 @@ Function *ShaderMerger::generateEsGsEntryPoint(Function *esEntryPoint, Function
   //     %gsPrimCount = call i32 @llvm.amdgcn.ubfe.i32(i32 %sgpr3, i32 8, i32 8)
   //
   //     %esEnable = icmp ult i32 %threadId, %esVertCount
-  //     br i1 %esEnable, label %.begines, label %.endes
+  //     br i1 %esEnable, label %.begins, label %.endes
```

Comment:
This should be dropped. begines, not begins. Of course, we should use camelcase in the future. begines -> beginEs. This is another PR to fix it :) Following codes have the same mistakes.

Reply:
Aha! And I wondered why that looks so intentional, now seems obvious from the context.